src/components/molecules/OcBreadcrumb/OcBreadcrumb.spec.js

```diff
@@ -20,15 +20,49 @@ describe("OcBreadcrumb", () => {
     expect(wrapper.props().variation).toMatch("lead")
     expect(wrapper.classes()).toContain("oc-breadcrumb-lead")
     expect(wrapper).toMatchSnapshot()
-  }),
-    it("displays all items", () => {
-      const wrapper = shallowMount(Breadcrumb, {
-        propsData: {
-          items,
-        },
-      })
+  })
+  it("displays all items", () => {
+    const wrapper = shallowMount(Breadcrumb, {
+      propsData: {
+        items,
+      },
+    })
+
+    expect(wrapper.findAll(".oc-breadcrumb-list-item").length).toBe(4)
+    expect(wrapper.findAll("#oc-breadcrumb-contextmenu-trigger").length).toBe(0)
+    expect(wrapper).toMatchSnapshot()
+  })
+  it("displays context menu trigger", () => {
+    const wrapper = shallowMount(Breadcrumb, {
+      propsData: {
+        items,
+      },
+      slots: {
+        contextMenu: "Example item",
+      },
+    })
+
+    expect(wrapper.findAll("#oc-breadcrumb-contextmenu-trigger").length).toBe(1)
+  })
+  it("does not display context menu trigger when no slot given", () => {
+    const wrapper = shallowMount(Breadcrumb, {
+      propsData: {
+        items,
+      },
+    })
 
-      expect(wrapper.findAll(".oc-breadcrumb-list-item").length).toBe(4)
-      expect(wrapper).toMatchSnapshot()
+    expect(wrapper.findAll("#oc-breadcrumb-contextmenu-trigger").length).toBe(0)
+  })
+  it("does not display context menu trigger with only one item", () => {
```

Comment:
Test not needed if you think that my comment about the length check is a good idea.